ethereumetl/streaming/item_exporter_creator.py

```diff
@@ -23,29 +23,30 @@
 from blockchainetl.jobs.exporters.console_item_exporter import ConsoleItemExporter
 from blockchainetl.jobs.exporters.multi_item_exporter import MultiItemExporter
 
-DEFAULT_KAFKA_TOPIC_MAPPING = {
-    'block': 'ethereum_blocks',
-    'transaction': 'ethereum_transactions',
-    'log': 'ethereum_logs',
-    'token_transfer': 'ethereum_token_transfers',
-    # todo(shashank): migrate token_transfers_v2 to new topic
-    'token_transfer_v2': 'token_transfers_v2',
-    'trace': 'ethereum_traces',
-    'contract': 'ethereum_contracts',
-    'token': 'ethereum_tokens',
+CHAIN_ID_TOPIC_PREFIX_MAPPING = {
+    1: 'ethereum',
+    4: 'ethereum_rinkeby',
+    5: 'ethereum_goerli',
+    137: 'polygon',
+    80001: 'polygon_mumbai',
 }
 
-TESTNET_KAFKA_TOPIC_MAPPING = {
-    'block': 'ethereum_rinkeby_blocks',
-    'transaction': 'ethereum_rinkeby_transactions',
-    'log': 'ethereum_rinkeby_logs',
-    'token_transfer': 'ethereum_rinkeby_token_transfers',
-    # todo(shashank): migrate token_transfers_v2 to new topic
-    'token_transfer_v2': 'testnet_token_transfers_v2',
-    'trace': 'ethereum_rinkeby_traces',
-    'contract': 'ethereum_rinkeby_contracts',
-    'token': 'ethereum_rinkeby_tokens',
-}
+def get_kafka_topic_mapping(chain_id):
+    if chain_id not in CHAIN_ID_TOPIC_PREFIX_MAPPING:
+        raise ValueError('Unable to determine topic mapping for chain_id ' + chain_id)
+    topic_prefix = CHAIN_ID_TOPIC_PREFIX_MAPPING[chain_id]
+    return {
+        'block': topic_prefix + '_blocks',
+        'transaction': topic_prefix + '_transactions',
+        'log': topic_prefix + '_logs',
+        'token_transfer': topic_prefix + '_token_transfers',
```

Comment:
@jdilla52 `token_transfers_v2` did not follow the naming convention completely as it was using `token_transfers_v2` and `testnet_token_transfers_v2` as topics instead of 
`ethereum_ token_transfers_v2` and `ethereum_goerli_token_transfers_v2`. Is there a reliable way to do this renaming ideally without data loss?

Reply:
there shouldn't be any technically. kafka should buffer out the message as we switch the consumers. In theory should be fine.

Reply:
we should ideally move the naming to lead with schema and then descriptor: dtoken_transfers_v2_testnet